spyder/api/translations.py

```diff
@@ -8,4 +8,4 @@
 This module contains the internationalization API.
 """
 
-from spyder.config.base import get_translation
+from spyder.config.base import _
```

Comment:
```suggestion
from spyder.config.base import _


get_translation = _
```

* This is necessary to not break API compatibility for external plugins that import `get_translation` from here. For instance, Spyder-line-profiler:

    https://github.com/spyder-ide/spyder-line-profiler/blob/master/spyder_line_profiler/spyder/plugin.py#L19

* I think that still preserves the singleton character of `_`, right?

Reply:
Ah, yes, that's a good point I hadn't thought of. I'll force-push an amended commit.